tox.ini

```diff
@@ -0,0 +1,70 @@
+[tox]
+envlist =
+    lint,
+    test
+
+[testenv]
+basepython = python3
+
+[testenv:test]
+deps =
+    pytest
+    pytest-girder
+commands =
+    pytest {posargs}
+
+[testenv:lint]
+skipsdist = true
+skip_install = true
+deps =
+    flake8
+    flake8-bugbear
+    flake8-docstrings
+    flake8-quotes
```

Comment:
If we want to use black, add `flake8-black`, and, similarly for `flake8-isort`.

Reply:
Haven't used `flake8-isort` before; do you find it useful/valuable?

Reply:
See 19118408453d7a5ccbceffdfd146f362c8c32418

Reply:
isort forces imports to be in a particular order.  Brian has added it to a number of projects.  It is mostly useful in that it can automatically reorder imports.  It is like using black -- nice in that it removes choice, bad in that it removes choice.